checks.nix

```diff
@@ -1,10 +1,17 @@
-{ packages, inputs, system }:
+{
+  packages,
+  inputs,
+  system,
+}:
 {
   pre-commit-check = inputs.pre-commit-hooks.lib.${system}.run {
     src = ./.;
     hooks = {
-      nixpkgs-fmt.enable = true;
+      alejandra.enable = true;
```

Comment:
Is there any option to run these outside of the pre-commit hooks? I find using pre-commit hooks unintuitive, and I'm not sure if they run if we don't run git inside `nix develop`.

Reply:
Yeah, they are provided by the shell as well so you can run `alejandra . ` or `nix-linter -r .` inside the nix shell